nuss-operator-api/swagger.yaml

```diff
@@ -2031,8 +2059,8 @@ definitions:
         format: "int32"
         description: "Number of satellites used in calculating the altitude_gps_wgs84_ft."
       enroute_positions_id:
-        type: "integer"
-        format: "int32"
+        type: "string"
+        format: "uuid"
```

Comment:
for swagger is this not $uuid?

Reply:
I see it as uuid utmdomain. Could you confirm.

Reply:
arg pardon its fine